Explain the custom polling (#31265)
home-assistant/home-assistant.github.io: source/_includes/common-tasks/define_custom_polling.md

If you want to define a specific interval at which your device is being polled for data, you can disable the default polling interval and create your own polling automation.

> The reason why we chose this approach is because this gives you way more flexibility on when to poll.

Reviewer: ```suggestion
Creating an automation for polling gives you more flexibility on when to poll.
```
What do you mean by *this approach*? Using an automation?

Author: Having users create an automation to customize polling yes

> 1. This already works with most of the integrations. Since changing the polling interval could be a use case for every integration, it makes sense to pick an approach that works for every integration without having to add support to every individual integration.

Reviewer: ```suggestion
1. Using the homassistant.update_entity service to update an entity already works with most of the integrations. Since changing the polling interval could be a use case for every integration, it makes sense to pick an approach that works for every integration without having to add support to every individual integration.
```
*This already works*... : What is *this* referring to?

Author: Using the `homassistant.update_entity` service to update an entity

> 1. This already works with most of the integrations. Since changing polling interval is something that could be a use case for ever integration, it makes sense to pick an approach that works for every integration without having to add support for every individual integrations.

Reviewer 2: ```suggestion
1. This already works with most of the integrations. Since changing the polling interval could be a use case for every integration, it makes sense to pick an approach that works for every integration without having to add support to every individual integration.
```

> 2. You can poll whenever you want. With this approach you can make your polling smarter and more efficient. For example, if you have a rate limited solar panel provider with a max amount of requests per day. It might make sense to lower/stop the polling at night so you can poll more when it actually makes sense.

Reviewer 2: ```suggestion
2. You can poll whenever you want. With this approach, you can make your polling smarter and more efficient. For example, if you have a rate-limited solar panel provider with a maximum number of requests per day, it might make sense to lower/stop the polling at night so you can poll more during the day when it matters.
```
